src/cli/index.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-console */
+import { execSync } from 'node:child_process'
+import fs from 'node:fs'
+import path from 'node:path'
+import process from 'node:process'
+import c from 'kleur'
+
+export async function cli() {
+  const cwd = process.cwd()
+
+  console.log(c.dim(`\nSetup eslint config...\n`))
+
+  // Update package.json
+  let pkg: Record<string, any>
+  const pkgPath: string = path.join(cwd, 'package.json')
+
+  try {
+    const pkgContent: string = fs.readFileSync(pkgPath, 'utf-8')
+    pkg = JSON.parse(pkgContent)
+  }
+  catch (error) {
+    console.log(c.red('Could not find -'), c.dim('package.json'))
+    process.exit(1)
+  }
+
+  const eslintLastVersion = await execSync(`npm show eslint version --quite`)
+  const eslintConfigLastVersion = await execSync(`npm show @antfu/eslint-config version --quite`)
+
+  const eslintVersion: string = eslintLastVersion?.toString()?.trim()
+  const eslintConfigVersion: string = eslintConfigLastVersion?.toString()?.trim()
+
+  pkg.type = 'module'
```

Comment:
This might be a bit dangerous. We could generate ESM/CJS based on the current preference, instead of directly changing it.

Reply:
Thank you, I hadnt thought of that at all